WordPressCom-Analytics-iOS.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "WordPressCom-Analytics-iOS"
-  s.version      = "0.0.36"
+  s.version      = "0.0.37"
```

Comment:
I'm confused.  The podspec in develop shows 0.0.36 https://github.com/wordpress-mobile/WordPressCom-Analytics-iOS/blob/develop/WordPressCom-Analytics-iOS.podspec
Were you thinking of the cocoapod version?